apisix/cli/ops.lua

```diff
@@ -916,10 +916,32 @@ local function stop(env)
 end
 
 
+local function sleep(n)
+  execute("sleep " .. tonumber(n))
+end
+
+
+local function wait_stop_finish(env, n)
+    for i = 1, n do
+         -- check running
+        local pid_path = env.apisix_home .. "/logs/nginx.pid"
+        local pid = util.read_file(pid_path)
+        pid = tonumber(pid)
+        if not pid then
+            print("old apisix process has stopped")
+            return
+        else
+            sleep(1)
+        end
+    end
+end
+
+
 local function restart(env)
   -- test configuration
   test(env)
   stop(env)
+  wait_stop_finish(env, 3)
```

Comment:
We have checked if the apisix is running in the function `start()`, can you simplify the logic that is repeated in these 2 functions `wait_stop_finish` and `start`

Reply:
ok